openbrokerapi/response.py

```diff
@@ -16,12 +15,16 @@ def __init__(self,
         self.description = description
 
 
+class AsyncResponse:
+    pass
+
```

Comment:
shows clearly, what could be async, that's nice. I always double checked with the spec ^^